app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -164,8 +170,20 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
     private fun initFindMeButton() {
         findMe = MapData("find_me")
         Tangram.addDataSource(findMe);
-        findMeButton?.visibility = View.VISIBLE
-        findMeButton?.setOnClickListener({ presenter?.onFindMeButtonClick() })
+        findMeButton.visibility = View.VISIBLE
+        findMeButton.setOnClickListener({ presenter?.onFindMeButtonClick() })
```

Comment:
I guess I'm unsure why in some places here you've changed from explicitly calling the setter functions for properties, and in others you haven't. Is this on purpose?